.stylelintrc

```diff
@@ -1,8 +1,196 @@
 {
-    "extends": "stylelint-config-standard",
-    "rules": {
-        "indentation": 4,
-        "declaration-empty-line-before": "never",
-        "number-leading-zero": "never"
-    }
+  "extends": "stylelint-config-standard",
+  "plugins": [
+    "stylelint-order"
+  ],
+  "rules": {
```

Comment:
Have we discussed the order properly? For me it makes sense to put border before background, because it can have an influence on element dimensions. I'd put stuff like colors to the end since they have no influence at layout.

But if this is some list others are using and we agreed on it, it's ok :)

Reply:
The rules are taken from: http://intranet/Wiki-Actum/Development/Front-end/Best-Practice/CSS-Guidelines